src/components/pages/SingleReview.jsx

```diff
@@ -21,20 +23,39 @@ const SingleReview = () => {
       });
   }, [review_id]);
 
+  useEffect(() => {
+    if (review) {
+      getCommentsByReviewId(review_id)
```

Comment:
Please use a loading state for the comments as well, so a message is displayed when they are loading.